### DISCUSSION TITLE
Update the text of a node base of the outcome of a formula

### DISCUSSION THREAD

**Discussion by sebastienrolle-eaton** [freeplane/freeplane#2104] [Help] [answered]
Hello, I am trying to customize the text in a node using the outcome of a formula. There is an attribute 'Wrok" in the node computed using a formula: =children.sum(0) { it['Work'].num0 }. Now to customize the name, i use Groovy like this 

node.text = node.attributes['Work'] But the computed value is not displayed; instead, the formula itself is shown!

Is Someone know of to get the computed value of a formula in Groovey ?

Thank

**Comment by Coffee-Search** [comment]
You can enter formulas to node attributes, node text, node notes, node descriptions directly. If you add the formula `=children.sum(0) { it['Work'].num0 }` to the node directly that will display the formula result. Or you can add the formula `="Work total: " + node['Work'].num0` to the node to reference it's own attribute.

**Reply by Coffee-Search** [reply, marked as answer]
... After thought - if you are trying to update the node text from a script - to access the calculated value of the formula don't use .attributes e.g. `node.text = node["Work"].num0`

**Reply by sebastienrolle-eaton** [reply]
Thank a lot. It is working fine. Have a good day.
